.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.4.2
+ruby-2.4.3
```

Comment:
Is it supposed to have the ruby- prefix in here?

Reply:
Yep